examples/gluon_alexnet/README.md

```diff
@@ -0,0 +1,219 @@
+4. [Load and serve a hybrid'ized gluon model](#load-and-serve-a-hybrid'ized-gluon-model)
+5. [How to test your gluon models](#how-to-test-your-gluon-models)
+6. [How to create a model archive](#how-to-create-a-model-archive)
+
+**NOTE: If you do not use `Symbols` parameter in `MANIFEST.json`, MMS infers this as a imperative gluon model
+(or network defined by run). In other words DO-NOT-USE `Symbols` parameter in `MANIFEST.json` when defining `Gluon` 
+models.** 
+
+## How to serve Gluon models
+There are three ways of defining our Gluon nerual network/model.
+1. Load and serve a pretrained model.
+2. Load and serve custom imperative model, which implies that we don't generate/use symbols.
+3. Load and serve custom Hybrid models. More on why we need `HybridBlocks` and the need to `hybridize` is explained well
+in this [Gluon Document](http://gluon.mxnet.io/chapter07_distributed-learning/hybridize.html#)
+
+For this tutorial, we use [Alexnet](https://mxnet.incubator.apache.org/api/python/gluon/model_zoo.html) 
+to describe the above three scenarios. The example code and supporting files to create a model archive are 
+in `mxnet-model-server/examples/gluon_alexnet/*`.  
+
+## Load and serve a pretrained gluon model
+Gluon already provides a good range of pretrained models through `mxnet.gluon.model_zoo` and they are rapidly adding 
+more. These models are available after simply installing `mxnet`
+```bash
+# For CPU instances
+pip install mxnet-mkl
+```
+
+This is the simplest of the three methods to define. Here we don't need `params` or `symbols` to be given as inputs. 
+We can access these models from the `service_code` as follows
+```text
+import mxnet
+
+class AlexnetService():
+...
+    def inference(self, data):
+        net = mxnet.gluon.model_zoo.vision.alexnet(pretrained=True)
+        output = net(data)
+        return output.argmax()
+...        
+```
+
+Since this is a pretrained model, we wouldn't need to provide `Parameters` and `Symbols` through `MANIFEST.json`.
+
+To see this in action, refer the `__init__` in the `MMSImperativeService`  in  `gluon_alexnet.py`. 
+ 
+Remove the `Parameters` and `Symbols` from the `MANIFEST.json` file and create a model-archive.
+Refer [How to test your models](#how-to-test-your-gluon-models), to test your gluon model.
+
+## Load and serve pure Gluon imperative models
+As above, let's consider `gluon_mxnet.py` in this `mxnet-model-server/examples/gluon_alexnet` folder.
+To define a imperative neral-net, we would need to define the complete network. We then load the parameters that we 
+obtained from training this model and start serving the model.
+
+```text
+import mxnet
+from mxnet import gluon
+from mxnet.gluon import nn
+
+class GluonImperativeAlexnet():
+    """
+    Fully imperative gluon Alexnet model
+    """
+    def __init__(self, classes=1000, **kwargs):
+        super(GluonImperativeAlexnet, self).__init__(**kwargs)
+        with self.name_scope():
```

Comment:
Is it possible to refer this to somewhere else where the network is defined. may be a gluon tutorial.